torchbiggraph/config.py

```diff
@@ -397,7 +397,10 @@ def __attrs_post_init__(self):
 
 
 # TODO make this a non-inplace operation
-def override_config_dict(config_dict: Any, overrides: List[str]) -> Any:
+def override_config_dict(config_dict: Any, overrides: Optional[List[str]]) -> Any:
```

Comment:
Since you're now doing the `chain` inside this function, the type of `overrides` is a nested list, namely `Optional[List[List[str]]]`. Same below.

Reply:
This one still applies